addons/base_address_extended/models/res_country.py

```diff
@@ -4,15 +4,10 @@
 from odoo import fields, models
 
 
-class ResCountry(models.Model):
+class Country(models.Model):
     _inherit = 'res.country'
 
-    street_format = fields.Text(
-        help="Format to use for streets belonging to this country.\n\n"
-             "You can use the python-style string pattern with all the fields of the street "
-             "(for example, use '%(street_name)s, %(street_number)s' if you want to display "
-             "the street name, followed by a comma and the house number)"
-             "\n%(street_name)s: the name of the street"
-             "\n%(street_number)s: the house number"
-             "\n%(street_number2)s: the door number",
-        default='%(street_number)s/%(street_number2)s %(street_name)s', required=True)
```

Comment:
> Unified street format to "Chaussee de Namur 40 - Appt 12"; the
configurable ones where actually wrong by country.

Wasn't it just missing implementations instead of being wrong?

Russia, Singapore, South Africa, South Corea have different formats for instance
https://bitboost.com/ref/international-address-formats.html

Reply:
@william-andre  But these countries don't need the street number to separated out in an EDI document - so the usual `street` field should suffice